app/views/devices_guidance/index.html.erb

```diff
@@ -29,9 +29,15 @@
       How to order
     </h2>
 
-    <p class="govuk-body govuk-!-margin-bottom-7">
-      <%= govuk_link_to "Sign in to order devices", sign_in_path, class: "app-emphasis-link" %>
-    </p>
+    <ul class="govuk-list govuk-list--spaced govuk-!-margin-bottom-7">
+      <li>
+        <%= govuk_link_to 'Get devices for state-funded pupils in independent special schools and alternative provision', devices_how_to_order_for_independent_special_schools_path %>
```

Comment:
Change to: How to get devices for state-funded pupils in independent special schools and alternative provision